DecipherNow/gm-fabric-dashboard
# 1003 apply uptime component to service list items

Resolves #1003 

## Proposed Changes

  - Implement Uptime component to Service instance list view uptime 
  
## How to Test

  - Fabric view > choose Go/Jvm service
  - Observe uptime updating every second under Uptime column
  
<img width="904" alt="screen shot 2017-12-06 at 1 01 52 pm" src="https://user-images.githubusercontent.com/26534838/33677100-aa63844c-da85-11e7-8b86-7d881bdf598d.png">
